README.md

```diff
@@ -13,10 +13,10 @@ first, to isolate the installed version and dependencies from your master enviro
 
 The following command will set up a conda virtual environment, add the necessary package channels, and install *magic-cta-pipe* and its dependencies::
 
-    git clone https://github.com/cta-observatory/magic-cta-pipe.git
+    git clone https://github.com/ranieremenezes/magic-cta-pipe.git
```

Comment:
You changed to your repo, but this actually should point to the repo in `cta-observatory` since you are merging on that repo. Next time, you can simply create a branch in this repo, no need to fork. Forks are ok, but for example they do not allow to run automatic tests if those tests require to access repository secrets (e.g. passwords for downloading test data):

Reply:
Thanks Alessio, I fixed the repository path and, after this PR, I will start working on a branch instead of the current fork.